application/application.bndrun

```diff
@@ -0,0 +1,58 @@
+-include: ../base.bndrun
+
+
+-runrequires.webconsole: \
```

Comment:
I was not able to find in any documentation `-runrequires.webconsole` notation.

There is a documentation about root bundle in `-runrequires`.

But there is nothing about prefix `-runrequires` , so I don't understand what `-runrequires.webconsole` is for.
Could you please explain this ?

Reply:
the `-runrequires` [documentation](https://bnd.bndtools.org/instructions/runrequires.html) holds filters to all main bundles that you needed. if you hit the resolve button or start resolving in maven it will generate the -runbundles automatically and finds all necessary dependencys.

so `-runbundles` could be removed from from this file because they will resolves via the resolve plugin. I often let them in the file to know what bundles changes (also an fail on change option in the plugin).

the suffix `.webconsole` or `.flow` is just to group the required bundles ans give the group a name.

Reply:
Well, I assumed that `.webconsole` is an identifier of something similar to `-runrequires`.
But what I don't understand is: how it's referenced ?

If I remove `-runrequires.flow`  then it seems nothing is broken. So these groups are never used ?
So why they are defined at all then ?
Apparently this is not a "reserved keyword" for bnd tool. So it has to be used in the config somewhere.
But I don't see this usage.